anker/src/instruction.rs

```diff
@@ -37,9 +39,16 @@ pub enum AnkerInstruction {
 
     /// Sell rewards to the UST reserve.
     SellRewards,
+    /// Transfer from the UST reserve to terra through Wormhole.
+    SendRewards {
+        nonce: u32, // FIXME: Check this nonce business.
+        fee: u64,   // FIXME: I don't know if this can be variable or not. Check it.
```

Comment:
=I asked Dave and he answered:
> Nonce is random
So it's fine to leave it caller's control.
> Fee is a portion of the tokens that will pay the relayer, e.g. if the sender doesn't have any tokens in the chain that it's sending to.
So we should definitely set this to 0, to avoid getting sniped by a malicious user that can act as a relayer and steal all the tokens 😅

Reply:
> So we should definitely set this to 0, to avoid getting sniped by a malicious user that can act as a relayer and steal all the tokens sweat_smile

Ah yes :fearful: we should hard-code it to 0 and not make it a parameter!